### ISSUE TITLE
MNT: Add CoC, License

### ISSUE THREAD

**Issue by pllim** [eteq/astrowidgets#2]
* Copy CoC from Astropy?
* BSD 3-clause license?

**Comment by mwcraig** [comment]
A BSD 3-clause has been added, so half way there...

**Comment by pllim** [comment]
@mwcraig , can we move the main license to root, so GitHub picks it up?

**Comment by mwcraig** [comment]
Yep, that makes sense. Do you want to or should I include it in the tests PR?

**Comment by pllim** [comment]
Go ahead, @mwcraig . I don't have time to work on this today anyway. Thanks!

**Comment by pllim** [comment]
OK just need CoC now. :)
